amazon-dcv-windows/README.md

```diff
@@ -0,0 +1,48 @@
+---
+display_name: Amazon DCV Windows
+description: Amazon DCV Server and Web Client for Windows
+icon: ../.icons/dcv.svg
+maintainer_github: coder
+partner_github: aws
+verified: true
+tags: [windows, dcv, web, desktop]
+---
+
+# Amazon DCV Windows
```

Comment:
For non AWS workspaces, does this work OK?

Reply:
Yes. I tested it on GCP, and it works. You will see a notification about getting the license, though.